src/main/resources/sponge_at.cfg

```diff
@@ -45,6 +45,9 @@ public net.minecraft.village.MerchantRecipe field_180323_f # rewardsExp
 
 public net.minecraft.block.state.BlockState$StateImplementation
 
+public-f net.minecraft.tileentity.TileEntity field_145855_i # nameToClassMap
```

Comment:
Why do you remove final from the field? You don't seem to be assigning new values to it.